packages/react-components/react-toast/src/components/Toast.tsx

```diff
@@ -84,13 +85,15 @@ export const Toast: React.FC<ToastProps & { visible: boolean }> = props => {
       toast.addEventListener('animationend', play, {
         once: true,
       });
-
-      return () => {
-        toast.removeEventListener('animationend', play);
-      };
     }
   }, [play, toastRef]);
 
+  React.useEffect(() => {
+    if (visible) {
+      announce(toastRef.current?.textContent ?? '', { politeness });
+    }
+  }, [announce, politeness, toastRef, visible, updateId]);
```

Comment:
This will run even `toastRef.current?.textContent` does not change. Is it intentional?

Reply:
yup screen, screen readers should only announce changed content